tasks_downstream.py

```diff
@@ -0,0 +1,580 @@
+        "trace": True,
+        "pathMapping": {},
+    }
+    if chrome_executable:
+        chrome_configuration["runtimeExecutable"] = chrome_executable
+    cw_config["launch"] = {
+        "compounds": [
+            {
+                "name": "Start Odoo and debug Python",
+                "configurations": ["Attach Python debugger to running container"],
+                "preLaunchTask": "Start Odoo in debug mode",
+            },
+            {
+                "name": "Run and debug Odoo tests",
+                "configurations": ["Attach Python debugger to running container"],
+                "preLaunchTask": "Run Odoo Tests in debug mode for current module",
+                "internalConsoleOptions": "openOnSessionStart",
+            },
+            {
+                "name": "Start Odoo and debug JS in Firefox",
+                "configurations": ["Connect to firefox debugger"],
+                "preLaunchTask": "Start Odoo",
+            },
+            {
+                "name": "Start Odoo and debug JS in Chrome",
+                "configurations": ["Connect to chrome debugger"],
+                "preLaunchTask": "Start Odoo",
+            },
+            {
+                "name": "Start Odoo and debug Python + JS in Firefox",
+                "configurations": [
+                    "Attach Python debugger to running container",
+                    "Connect to firefox debugger",
+                ],
+                "preLaunchTask": "Start Odoo in debug mode",
+            },
+            {
+                "name": "Start Odoo and debug Python + JS in Chrome",
+                "configurations": [
+                    "Attach Python debugger to running container",
+                    "Connect to chrome debugger",
+                ],
+                "preLaunchTask": "Start Odoo in debug mode",
+            },
+        ],
+        "configurations": [
+            debugpy_configuration,
+            firefox_configuration,
+            chrome_configuration,
+        ],
+    }
+    # Configure folders
+    cw_config["folders"] = []
+    for subrepo in SRC_PATH.glob("*"):
+        if not subrepo.is_dir():
+            continue
+        if (subrepo / ".git").exists() and subrepo.name != "odoo":
+            cw_config["folders"].append(
+                {"path": str(subrepo.relative_to(PROJECT_ROOT))}
+            )
+        debugpy_configuration["pathMappings"].append(
+            {
+                "localRoot": "${workspaceRoot:%s}" % subrepo.name,
+                "remoteRoot": f"/opt/odoo/custom/src/{subrepo.name}",
+            }
+        )
+        for addon in chain(subrepo.glob("*"), subrepo.glob("addons/*")):
+            if (addon / "__manifest__.py").is_file() or (
+                addon / "__openerp__.py"
+            ).is_file():
+                url = f"http://localhost:{ODOO_VERSION:.0f}069/{addon.name}/static/"
+                path = "${{ '{{' }}workspaceRoot:{}{{ '}}' }}/{}/static/".format(
```

Comment:
Is this something left from the jinja template?
```suggestion
                path = "${{workspaceRoot:{}}}/{}/static/".format(
```